docs/releases/1.10.txt

```diff
@@ -36,6 +36,9 @@ Minor features
   link <django.contrib.admin.AdminSite.site_url>` at the top of each admin page
   will now point to ``request.META['SCRIPT_NAME']`` if set, instead of ``/``.
 
+* The admin's ``ChangeList`` will always use ``root_queryset.count()`` if
```

Comment:
I'm not sure this needs to be documented (bug fix instead of new feature), but I think most users won't understand the reference to root_queryset.count() (I don't anyway).